Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -120,6 +113,54 @@ - (void)viewDidLoad
     [self refreshData];
 }
 
+- (void)setupNavigationItems
+{
+    self.titleButton = [UIButton buttonWithType:UIButtonTypeSystem];
+    [self.titleButton addTarget:self action:@selector(changeGroup:) forControlEvents:UIControlEventTouchUpInside];
+    self.titleButton.titleLabel.lineBreakMode = NSLineBreakByTruncatingMiddle;
+    self.titleButton.titleLabel.textAlignment = NSTextAlignmentCenter;
+    self.titleButton.titleLabel.numberOfLines = 1;
+
+    self.titleTipButton = [UIButton buttonWithType:UIButtonTypeSystem];
+    [self.titleTipButton addTarget:self action:@selector(changeGroup:) forControlEvents:UIControlEventTouchUpInside];
+    self.titleTipButton.titleLabel.textAlignment = NSTextAlignmentCenter;
+    self.titleTipButton.titleLabel.lineBreakMode = NSLineBreakByTruncatingMiddle;
+    self.titleTipButton.titleLabel.numberOfLines = 1;
+    NSString *localizedOptionHint = NSLocalizedString(@"Tap here to change", "Tip for tapping media picker title to change the group.");
+    NSString *callForAction = [NSString stringWithFormat:@"%@ %@",localizedOptionHint, ArrowDown];
+    UIFont *titleFont = self.titleButton.titleLabel.font;
+    NSMutableAttributedString *titleTip = [[NSAttributedString alloc] initWithString:callForAction attributes:@{NSFontAttributeName: [titleFont fontWithSize:floorf(titleFont.pointSize * 0.75)]}];
```

Comment:
I don't think we need to construct an attributed string here – can't we just set the font on the button's titleLabel directly, now that they're separate?